services-custom/dynamodb-enhanced/README.md

```diff
@@ -80,66 +80,109 @@ values used are also completely arbitrary.
  
 ### Common primitive operations
 These all strongly map to the primitive DynamoDB operations they are
-named after. These examples are the most simple variants of each
-operation possible. These commands can be customized by using the
-builders provided for each command and offer most of the features
-available in the low-level DynamoDB SDK client.
+named after. The examples below are the most simple variants of each
+operation possible, using the the two styles available for constructing
+requests with either builder or consumers. These commands can be 
+customized by using the builders provided for each command and offer 
+most of the features available in the low-level DynamoDB SDK client.
 
    ```java
    // CreateTable
-   customerTable.createTable(CreateTableEnhancedRequest.create());
+   customerTable.createTable(() -> {});
```

Comment:
I'm not sure I like this, speaking to Matt he said there is prior art for having an overloaded version of this that takes no arguments at all. So-called 'simple' method form.